app/src/main/java/swati4star/createpdf/util/ExtractImages.java

```diff
@@ -23,11 +23,28 @@
     private ExtractImagesListener mExtractImagesListener;
     private int mImagesCount = 0;
     private ArrayList<String> mOutputFilePaths;
+    private PDFEncryptionUtility mPdfEncryptionUtility;
+    private PDFUtils mPDFUtils;
 
-    public ExtractImages(String mPath, ExtractImagesListener mExtractImagesListener) {
+    public ExtractImages(String mPath, PDFEncryptionUtility pdfEncryptionUtility,
+                         PDFUtils pdfUtils, ExtractImagesListener mExtractImagesListener) {
```

Comment:
we shouldnt take `pdfUtils` or `pdfEncryptionUtility` as parameters to this function. we can create instance of these classes here only. Just take "Context" as the paramter